README.md

```diff
@@ -1,34 +1,33 @@
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
-[![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/se-edu/addressbook-level4?branch=master)
+<!---[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/CS2103AUG2016-T09-C2/main)
+[![Coverage Status](https://coveralls.io/repos/github/se-edu/addressbook-level4/badge.svg?branch=master)](https://coveralls.io/github/CS2103AUG2016-T09-C2/main?branch=master)
+-->
 
-# Address Book (Level 4)
+# Jimi, the task manager for people like Jim.
+
+In our daily lives, we often find ourselves striving to be as successful as we can in whatever we do - be it in our academic pursuits or in our career choices. So naturally, this begs the question, how does one achieve success? Quite a tough question that many ask themselves. We can, however, attempt to answer a question like this by drawing parallels between our lives and the lives of successful individuals. Interestingly enough, all these success stories tend to share a common theme. They all seem to gravitate towards one key secret to success: **to always have a plan, be on track and be organized.**
+
+This provided the impetus for our team to develop a task managing app. Now, you're probably thinking to yourself, "Why would I need another task manager like this when I can pick from thousands of other competing products, in the process, entirely destroying your business model and extinguishing any pathetic desires you have of ever making it in the field of software engineering?" 
+<br><br><br>
+Well, you're right... you don't. **NOT UNLESS YOU'RE SOMEONE LIKE** [**JIM.**](http://www.comp.nus.edu.sg/~cs2103/AY1617S1/contents/handbook.html#handbook-project-product) <br><br>
```

Comment:
Can you improve the mock up GUI a bit more so I can see how each task is shown and what details of a task is visible. i.e. at time t what would Jim's task manager look like